File: Java/Lista 4/L4_Ex1.1/src/l4_ex1/pkg1/UsuarioAdmin.java

package l4_ex1.pkg1;

import java.util.Scanner;

public class UsuarioAdmin extends Usuario {
    
    public UsuarioAdmin(String login) {
        super(login);
    }
    
    @Override
    public int menu(){
        int opcao = 0;
        Scanner ler = new Scanner( System.in );
        System.out.println("----------MENU---------");
        System.out.println("(1)listarAutores");
        System.out.println("(2)contarCiencia");
        System.out.println("(3)listarFilosofia");
        System.out.println("(4)listarTudo");
        System.out.println("(5)inserirLivro");
        System.out.println("(6)inserirEstante");
        System.out.println("(7)removerLivro");
        System.out.println("");
        System.out.printf("Digite uma opcao:");
        
        opcao=ler.nextInt();
        
        if(opcao<1 || opcao>7){
            opcao=0;
        }        
        return opcao;
    }
    
}

File: Swift/L4/Lista_04_EX01/UsuarioAdmin.swift
Public class UsuarioAdmin : Usuario {
    
    Override func int menu(){
        var opcao = 0
        
        println("----------MENU---------");
        println("(1)listarAutores");
        println("(2)contarCiencia");
        println("(3)listarFilosofia");
        println("(4)listarTudo");
        println("(5)inserirLivro");
        println("(6)inserirEstante");
        println("(7)removerLivro");
        println("");
        printf("Digite uma opcao:");
        
        if(opcao<1 || opcao>7){
            opcao=0
        }        
        return opcao
    }
    
}